Modules/Scripted/SegmentStatistics/SegmentStatisticsCalculators/ClosedSurfaceSegmentStatisticsCalculator.py

```diff
@@ -0,0 +1,72 @@
+import vtk
+import slicer
+from SegmentStatisticsCalculators import SegmentStatisticsCalculatorBase
+
+
+class ClosedSurfaceSegmentStatisticsCalculator(SegmentStatisticsCalculatorBase):
+  """Statistical calculator for closed surfaces"""
+
+  def __init__(self):
+    super(ClosedSurfaceSegmentStatisticsCalculator,self).__init__()
+    self.name = "Closed Surface"
+    self.keys = tuple(self.name+'.'+m for m in ("surface_mm2", "volume_mm3", "volume_cc"))
+    self.defaultKeys = self.keys # calculate all measurements by default
+    #... developer may add extra options to configure other parameters
+
+  def computeStatistics(self, segmentID):
+    import vtkSegmentationCorePython as vtkSegmentationCore
+    requestedKeys = self.getRequestedKeys()
+
+    segmentationNode = slicer.mrmlScene.GetNodeByID(self.getParameterNode().GetParameter("Segmentation"))
+
+    if len(requestedKeys)==0:
+      return {}
+
+    containsClosedSurfaceRepresentation = segmentationNode.GetSegmentation().ContainsRepresentation(
+      vtkSegmentationCore.vtkSegmentationConverter.GetSegmentationClosedSurfaceRepresentationName())
+    if not containsClosedSurfaceRepresentation:
+      return {}
+
+    segment = segmentationNode.GetSegmentation().GetSegment(segmentID)
+    closedSurfaceName = vtkSegmentationCore.vtkSegmentationConverter.GetSegmentationClosedSurfaceRepresentationName()
+    segmentClosedSurface = segment.GetRepresentation(closedSurfaceName)
+
+    # Compute statistics
+    massProperties = vtk.vtkMassProperties()
+    massProperties.SetInputData(segmentClosedSurface)
+
+    # Add data to statistics list
+    ccPerCubicMM = 0.001
+    stats = {}
+    if "Closed Surface.surface_mm2" in requestedKeys:
+      stats["Closed Surface.surface_mm2"] = massProperties.GetSurfaceArea()
+    if "Closed Surface.volume_mm3" in requestedKeys:
+      stats["Closed Surface.volume_mm3"] = massProperties.GetVolume()
```

Comment:
You are mixing human-readable string (Closed Surface) that may need to be translated, contain international characters, etc., and machine code (volume_mm3) that uses restricted character set (it shouldn't contain your separator character, or other special characters).
Either use the class name to distinguish plugins; or have a separate plugin name and title attributes (name: machine-readable, such as "ClosedSurface" or "closed_surface"; title: human-readable, such as "Closed surface").